src/hivpy/sexual_behaviour.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+
+from . import sex_behaviour_data as sb
+
+
+class SexualBehaviourModule:
+
+    def __init__(self, **kwargs):
+        # Randomly initialise sexual behaviour group transitions
+        self.sex_behaviour_trans_male = np.random.choice(sb.sex_behaviour_trans_male_options)
+        self.sex_behaviour_trans_female = np.random.choice(sb.sex_behaviour_trans_female_options)
```

Comment:
Maybe combining these in one would simplify things a bit (here and when they're accessed)? For example, a dictionary where the key is male or female. Not sure though.
(and similarly for the other attributes)